rules/windows/impact_volume_shadow_copy_deletion_via_wmic.toml

```diff
@@ -14,6 +14,56 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Volume Shadow Copy Deletion via WMIC"
+note = """## Triage and analysis
+
+### Investigating Volume Shadow Copy Deletion via WMIC
+
+The Volume Shadow Copy Service (VSS) is a Windows feature that enables system administrators to take snapshots of volumes
+that can later be restored or mounted to recover specific files or folders.
+
+A typical step in the playbook of an attacker attempting to deploy ransomware is to delete Volume Shadow
+Copies to ensure that victims have no alternative to paying the ransom, making any action that deletes shadow
+copies worth monitoring.
+
+This rule monitors the execution of wmic.exe to interact with VSS via the shadowcopy alias and delete parameter.
```

Comment:
```suggestion
This rule monitors the execution of `wmic.exe` to interact with VSS via the shadow copy alias and delete parameter.
```

Reply:
Style suggestion and separated "shadow copy" because it looks to be written as two separate words throughout the rule docs. Ignore if it should be a single word in this case!

Reply:
shadowcopy is an wmic alias, editing it to apply the style suggestion.